src/cclib/parser/daltonparser.py

```diff
@@ -501,6 +515,9 @@ def extract(self, inputfile, line):
             self.set_attribute("charge", int(line.split()[-1]))
         if "@    Spin multiplicity and 2 M_S" in line:
             self.set_attribute("mult", int(line.split()[-2]))
+        # Dalton only has ROHF, no UHF
+            if self.mult != 1:
+                self.metadata["spintype"] = "ROHF"
```

Comment:
I'd suggest some sort of attribute for R, RO, U that's distinct from method. Some programs, for example, call it ROB3LYP or UB3LYP. If not spin type, then something like "open shell"?

But I definitely agree "method" should include all the methods. Consider a composite method like G3 or other thermochemical recipe - there will be multiple methods.